awx/conf/migrations/0009_rename_proot_settings.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from django.db import migrations
+from awx.conf.migrations import _rename_setting
+
+
+def rename_proot_settings(apps, schema_editor):
+    _rename_setting.rename_setting(apps, schema_editor, old_key='AWX_PROOT_BASE_PATH', new_key='AWX_ISOLATION_BASE_PATH')
```

Comment:
I don't think we actually use this anywhere. Can we just delete this setting?

Reply:
Never mind... I read the rest of the PR.

But now I'm wondering if this can get rolled into the isolated code removal and we just ditch this setting all together. I guess it doesn't really matter.

Reply:
Ignoring isolated nodes, we *do* still lay down a private data dir on the file system (in `/tmp`) *before* jobs run, right?